src/collective/folderorder/testing.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+"""Setup testing infrastructure."""
+from plone.app.testing import FunctionalTesting
+from plone.app.testing import IntegrationTesting
+from plone.app.testing import PloneSandboxLayer
+
+import pkg_resources
+
+
+try:
+    pkg_resources.get_distribution('plone.app.contenttypes')
+except pkg_resources.DistributionNotFound:
+    from plone.app.testing import PLONE_FIXTURE
+else:
+    from plone.app.contenttypes.testing import PLONE_APP_CONTENTTYPES_FIXTURE as PLONE_FIXTURE
```

Comment:
Is this line because of https://stackoverflow.com/questions/35509829/automatically-install-default-content-types-in-tests-on-plone-5? A small doc would be nice.

Reply:
yes, is the way we're using it all over the place.